charts/verdaccio/templates/_helpers.tpl

```diff
@@ -3,17 +3,54 @@
 Expand the name of the chart.
 */}}
 {{- define "verdaccio.name" -}}
-{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
-{{- end -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" }}
+{{- end }}
 
 {{/*
 Create a default fully qualified app name.
 We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+If release name contains chart name it will be used as a full name.
 */}}
 {{- define "verdaccio.fullname" -}}
-{{- $name := default .Chart.Name .Values.nameOverride -}}
-{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
-{{- end -}}
+{{- if .Values.fullnameOverride }}
```

Comment:
Is this a new value? `Values.fullnameOverride` is not in the README or `values.yaml`

Reply:
Yes, this is a new value, will update the Readme accordingly

Reply:
Just pushed [`eded0fd`](https://github.com/verdaccio/charts/pull/53/commits/eded0fd94cbc6b5132bf859d6336c516d15017b8) which fixes the typo above and adds `fullnameOverride` and `nameOverride` to Readme